srcpkgs/luarocks-lua53/template

```diff
@@ -18,7 +18,7 @@ maintainer="Duncaen <duncaen@voidlinux.org>"
 license="MIT"
 homepage="https://luarocks.org/"
```

Comment:
```suggestion
homepage="https://luarocks.org/"
changelog="https://raw.githubusercontent.com/luarocks/luarocks/master/CHANGELOG.md"
```

Reply:
Maybe https://github.com/luarocks/luarocks/wiki/Release-history is a better link since it contains the 3.9.0 and 3.9.1 releases, while CHANGELOG.md is outdated with 3.8.0?

Reply:
works for me